<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeadTracker</class>
 <widget class="QMainWindow" name="HeadTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HeadTracker</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHeadTracker">
    <property name="title">
     <string>HeadTracker</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionRecord"/>
   </widget>
   <addaction name="menuHeadTracker"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="headtracker.qrc">
     <normaloff>:/images/application-exit.png</normaloff>:/images/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="headtracker.qrc">
     <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="headtracker.qrc">
     <normaloff>:/images/media-playback-stop.png</normaloff>:/images/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="headtracker.qrc">
     <normaloff>:/images/media-record.png</normaloff>:/images/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="headtracker.qrc"/>
 </resources>
 <connections/>
</ui>
